gamemodes/zombiesurvival/gamemode/init.lua

```diff
@@ -3295,7 +3302,7 @@ function GM:KeyPress(pl, key)
 			else
 				local plvel = pl:GetVelocity()
 				if pl:GetPhysicsObject():IsPenetrating() then
-					if plvel == vector_origin then
+					if plvel == vector_origin or playerIsStuckInsideObj(plvel) then
```

Comment:
if plvel == vector_origin or plvel.x == 0 and plvel.y == 0 and plvel.z == -4.5 then

Reply:
Cleaner:
if plvel.x == 0 and plvel.y == 0 and (plvel.z == 0 or plvel.z == -4.5) then